diff --git a/lib/dataconnection.ts b/lib/dataconnection.ts
index 7b72764..2170bb3 100644
--- a/lib/dataconnection.ts
+++ b/lib/dataconnection.ts
@@ -61,7 +61,7 @@ export class DataConnection extends BaseConnection implements IDataConnection {
 			DataConnection.ID_PREFIX + util.randomToken();
 
 		this.label = this.options.label || this.connectionId;
-		this.serialization = this.options.serialization || SerializationType.Binary;
+		this.serialization = this.options.serialization || SerializationType.JSON;
 		this.reliable = !!this.options.reliable;
 
 		this._encodingQueue.on("done", (ab: ArrayBuffer) => {
diff --git a/lib/peer.ts b/lib/peer.ts
index 60e4956..46e6894 100644
--- a/lib/peer.ts
+++ b/lib/peer.ts
@@ -170,11 +170,6 @@ export class Peer extends EventEmitter<PeerEvents> {
 		};
 		this._options = options;
 
-		// Detect relative URL host.
-		if (this._options.host === "/") {
-			this._options.host = window.location.hostname;
-		}
-
 		// Set path correctly.
 		if (this._options.path) {
 			if (this._options.path[0] !== "/") {
diff --git a/lib/supports.ts b/lib/supports.ts
index 902b61a..5c59522 100644
--- a/lib/supports.ts
+++ b/lib/supports.ts
@@ -1,9 +1,3 @@
-import webRTCAdapter_import from "webrtc-adapter";
-
-const webRTCAdapter: typeof webRTCAdapter_import =
-	//@ts-ignore
-	webRTCAdapter_import.default || webRTCAdapter_import;
-
 export const Supports = new (class {
 	readonly isIOS = ["iPad", "iPhone", "iPod"].includes(navigator.platform);
 	readonly supportedBrowsers = ["firefox", "chrome", "safari"];
@@ -33,40 +27,15 @@ export const Supports = new (class {
 	}
 
 	getBrowser(): string {
-		return webRTCAdapter.browserDetails.browser;
+		return "chrome";
 	}
 
 	getVersion(): number {
-		return webRTCAdapter.browserDetails.version || 0;
+		return this.minChromeVersion;
 	}
 
 	isUnifiedPlanSupported(): boolean {
-		const browser = this.getBrowser();
-		const version = webRTCAdapter.browserDetails.version || 0;
-
-		if (browser === "chrome" && version < this.minChromeVersion) return false;
-		if (browser === "firefox" && version >= this.minFirefoxVersion) return true;
-		if (
-			!window.RTCRtpTransceiver ||
-			!("currentDirection" in RTCRtpTransceiver.prototype)
-		)
-			return false;
-
-		let tempPc: RTCPeerConnection;
-		let supported = false;
-
-		try {
-			tempPc = new RTCPeerConnection();
-			tempPc.addTransceiver("audio");
-			supported = true;
-		} catch (e) {
-		} finally {
-			if (tempPc) {
-				tempPc.close();
-			}
-		}
-
-		return supported;
+		return false;
 	}
 
 	toString(): string {
